hipe4ml/model_handler.py

```diff
@@ -358,6 +358,11 @@ def optimize_params_optuna(self, data, hyperparams_ranges, cross_val_scoring, nf
             A string indicating the filename of the study. If None,
             the study is not saved into a file.
 
+        early_stopping: int
+            Minimum improvement to continue the optimization. The
+            study will be stopped if no improvement is observed
+            within 10 iterations.
+
```

Comment:
I think the delicate point in the early stopping is how to set this number, which depends on the metric chosen and the particular problem. Maybe some more details on this should be provided.